docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
-  described when using assistive technologies.
-- enhance the details component to help users of assistive technologies
-  understand whether it is expanded or collapsed, and to make the component
-  behave correctly for users of Internet Explorer 8.
+To make GOV.UK Frontend’s images and font work in your project, you can either:
 
-You should [include](#option-1-include-javascript) or [import](#option-2-import-javascript) GOV.UK Frontend JavaScript, and then initialise the script in your application to ensure that all users can use it successfully.
+- route requests to the files in GOV.UK Frontend's folders - recommended
+- copy GOV.UK Frontend's files into your project folder
 
-Note that GOV.UK Frontend does not initialise any scripts by default; all scripts **must** be initialised in order for them to work.
+#### Route requests - recommended
 
-### Option 1: Include JavaScript
+We recommend you route requests for files in `/assets` to the `/node_modules/govuk-frontend/govuk/assets` folder.
 
-Include the `node_modules/govuk-frontend/govuk/all.js` script on your page. You might wish to copy the file into your project or reference it from `node_modules`.
+For example if you're using [express.js](https://expressjs.com/), add the following to your `app.js` file:
 
-To initialise all components, use the `initAll` function.
-
-JavaScript in GOV.UK Frontend requires HTML to be parsed first by the browser before it is initialised. Because of this, make sure you include the script before the closing `</body>` tag.
-Including the script elsewhere will stop components from functioning or displaying correctly.
-
-```html
-    <script src="path-to-assets/govuk-frontend/govuk/all.js"></script>
-    <script>window.GOVUKFrontend.initAll()</script>
-  </body>
-</html>
+```javascript
+var path = require('path');
+app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
 ```
 
-#### Initialise GOV.UK Frontend in only certain sections of a page
-
-By default, the `initAll` function from GOV.UK Frontend initialises all components scoped to an entire page with the `document` object.
+#### Copy files
 
-You can change this by passing the `scope` parameter to the `initAll` function.
+Copy the following folders into your project folder:
 
-For example, if you have a modal dialog box that opens with new markup you could do the following:
-
-```js
-var $modal = document.querySelector('.modal')
-window.GOVUKFrontend.initAll({
-  scope: $modal
-})
-```
+- `/node_modules/govuk-frontend/govuk/assets/images`
+- `/node_modules/govuk-frontend/govuk/assets/fonts`
 
-#### Initialise individual included components
+You should use an automated task or your build pipeline to copy the files, so your project folder stays up to date when we update GOV.UK Frontend.
 
-GOV.UK Frontend components with JavaScript behaviour have the `data-module` attribute set in their markup.
+#### If you use different folders
```

Comment:
I think that this section is a subsection [Copy files](https://github.com/alphagov/govuk-frontend/pull/1697/files#diff-0eb006911fb60032aa38031190ad4869R80).

Reply:
Ah good point, thanks. This takes us down to an h5, which is probably getting a bit much... may need to rejig after research.